server/backend/users/tests/test_serializers_user.py

```
@@ -0,0 +1,43 @@
+        assert serializer.data['is_superuser'] == user.is_superuser
+
+    def test_instructor_serializer(self):
+        user = User.objects.create_user(username='instructor1', password='testpass123', role='instructor', email='instructor1@example.com')
+        instructor = Instructor.objects.get(user=user)
+        serializer = InstructorSerializer(instructor)
+        assert serializer.data['id'] == instructor.id
+        assert serializer.data['name'] == user.username
+        assert serializer.data['designation'] == instructor.designation
+        assert serializer.data['university'] == instructor.university
+
+    def test_instructor_serializer_missing_user(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_instructor_serializer_missing_user`

Add return type annotation: `None`

(ANN201)